test/Microsoft.Azure.Relay.UnitTests/Microsoft.Azure.Relay.UnitTests.csproj

```diff
@@ -14,8 +14,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.PlatformAbstractions" Version="1.1.0" />
```

Comment:
>PlatformAbstractions [](start = 52, length = 20)

This was returning values with incorrect version numbers in them. Presumably since it's from 2016.